internal/component/cloudflare/workers/workers.go

```diff
@@ -0,0 +1,395 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package workers contains a base class that is used by components that are based on Cloudflare Workers.
+package workers
+
+import (
+	"bytes"
+	"context"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"net/textproto"
+	"strconv"
+	"strings"
+	"time"
+
+	cfworkerscode "github.com/dapr/components-contrib/internal/component/cloudflare/workers/code"
+	"github.com/dapr/kit/logger"
+	"github.com/dapr/kit/ptr"
+)
+
+const (
+	// Minimum version required for the running Worker.
+	minWorkerVersion = 20221209
+	// Issuer for JWTs
+	tokenIssuer = "dapr.io/cloudflare" //nolint:gosec
+	// JWT token expiration
+	tokenExpiration = 5 * time.Minute
```

Comment:
Do we want these to be configurable? And for default should we place defaults in metadata?

Reply:
I don't think this specific value should be configurable. The component is generating a new JWT on every request, which are considered disposable.

Actually, I may reduce this to 1 min for better security.